lib/checkcondition.cpp

```diff
@@ -862,6 +862,28 @@ static std::string conditionString(bool not1, const Token *expr1, const std::str
            (expr1->isName() ? expr1->str() : std::string("EXPR"));
 }
 
+static std::string conditionString(const Token * tok)
```

Comment:
how is this `conditionString` important in the checker logic?

If you think that the `Token::expressionString()` output is ugly then feel free to update it.

Reply:
Well, `expressionString` is kind of ugly since it adds no spaces, but thats not the main issue. More cases are now caught by the `isOppositeCond`, which the output message are different. This is to make it consistent with the original message, which also reverses some conditions(ie `x > 1` instead of `1 < x`).

Reply:
Also, changing `expressionString` would require changing a lot more test cases because the test string would be different. I am trying to keep the test cases that need to be updated to a minimum.